.github/workflows/changelog.yml

```
@@ -0,0 +1,250 @@
+name: Changelog Updater
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)